### ISSUE TITLE
The command "eval yarn" failed for node 4

### ISSUE THREAD

**Issue by tony19** [travis-ci/travis-ci#7119]
For some reason, my Travis [job](https://travis-ci.org/tony19/polymer-typescript-starter-kit/jobs/190123398) keeps erroring out on Node 4 builds with:

    The command "eval yarn" failed. Retrying, 2 of 3.

The Travis config doesn't include installing `yarn`, so I don't understand where the error is coming from. How can I fix this?

**Comment by tony19** [comment]
Right, I expect `yarn` to run successfully on Travis, given the bot is for Node 4, which is listed as supported in the Travis docs. Why does the command fail?

**Comment by BanzaiMan** [comment]
It's in the logs.

https://travis-ci.org/tony19/polymer-typescript-starter-kit/jobs/190123398#L299-L300

```
error gulp-html-lint@0.0.1: The engine "node" is incompatible with this module. Expected version ">=5.0.0".
error Found incompatible module
```

**Comment by tony19** [comment]
Ok, got it now :) Thanks!

**Comment by maulberto3** [comment]
Hi, I am getting a similar issue. Right before `the command eval yarn failed. Retrying, 2 of 3.` log, I have ```
$ npm run build
error Couldn't find the binary npm run build
``` log. However, inside my package,json I clearly have ```
"(...)
scripts": {
    "prepublish": "npm run build",
    "build": "babel ./src --out-dir ./dist --ignore __tests__",
    "pretest": "eslint ./src",
    "test": "jest",
    "demo": "browserify -t babelify ./demo/index.js -o ./demo/build.js",
    "start": "node server"
  }, (...)
```

**Comment by maulberto3** [comment]
Other threads say using newer node versions fixes the issue. Modified travis.yaml to use node 8. Will report soon.
